test/test_distributed.py

```diff
@@ -1176,6 +1176,18 @@ def _test_DDP_2iter(
             # Shuffle the input so that DDP input is different
             input = input[torch.randperm(batch_size)]
 
+        # Test that saving and loading work
+        # TODO: It should be possible to save the entire model,
+        # but this doesn't work at the moment.  Update this test
+        # when it does work.
+        tmp_file = tempfile.TemporaryFile()
```

Comment:
This should be in a with clause or otherwise you’re leaking files

Reply:
This doesn't seem to match the documentation from Python:

> Return a file-like object that can be used as a temporary storage area. The file is created using mkstemp(). It will be destroyed as soon as it is closed (including an implicit close when the object is garbage collected).

Reply:
Furthermore, `TemporaryFile` is unlinked on POSIX, so it doubly can't be leaked.